larq/models.py

```diff
@@ -91,3 +107,10 @@ def summary(model, tablefmt="simple", print_fn=None):
     print_fn(f"Total params: {trainable_count + non_trainable_count}")
     print_fn(f"Trainable params: {trainable_count}")
     print_fn(f"Non-trainable params: {non_trainable_count}")
+
+    float32_equiv = (amount_binarized + amount_full_precision) * 0.125 * 32 / 1000000
+    compression_ratio = total_memory / (float32_equiv)
+
+    print_fn("Float32 Equivalent: " + str(float32_equiv) + " Mb")
```

Comment:
Does this handle rounding correctly? If not, I think the following will do:
```suggestion
    print_fn(f"Float-32 Equivalent: {float32_equiv:.2f} MB")
```